demo/src/cost_estimator.cpp

```diff
@@ -0,0 +1,55 @@
+#include "demo/cost_estimator.hpp"
+
+namespace demo {
+
+double CostEstimator::estimateCost(const Command& command, bool /*isActive*/) {
+    Positions absolutePath = toAbsolutePath(command.path);
+
+    int dotCounter = dotsAlongPath(absolutePath) + dotsInRadius(absolutePath.back());
+
+    int pathLength = static_cast<int>(absolutePath.size());
+    int neighborhoodSize = static_cast<int>(std::pow(2 * parameters_.pathEndNeighborhoodRadius + 1, 2));
+    int numOfCells = pathLength + neighborhoodSize;
+
+    return static_cast<double>(numOfCells) / dotCounter;
```

Comment:
We should handle the case of 0 dots here. It shouldn't ever occur but still.

My proposal would be to just return `std::numeric_limits<double>::max()` in that case. An alternative would be to throw an exception but setting the cost to "infinite" seems more reasonable to me.

Reply:
I implemented it that way in [80e05f9](https://github.com/KIT-MRT/arbitration_graphs/pull/38/commits/80e05f90544ed9c1b7b346470d0b45d46f73b319)